### PR TITLE
Add Vue.js demo

### PR DESCRIPTION
**Pull request by mnicpt**

### CODE REVIEW

File: demo/vue.htm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (gregjopa) [inline demo/vue.htm:6]:
```suggestion
  <title>Vue Demo - PayPal JS SDK</title>
```

**Comment by gregjopa** [comment]
This looks great! I'm 👍 with merging. Verified locally:

Screen Shot 2020-06-23 at 2 46 03 PM

Can we delete the babel polyfill code? (`https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.38/browser.js`) I deleted it locally and everything works fine in all the browsers installed on my Mac.

**Comment by mnicpt** [comment]
@gregjopa I'm guessing that it was cached.  When I removed it, the buttons didn't show up until I added it back.

**Comment by gregjopa** [comment]
@mnicpt Here's the change I made to remove babel: https://github.com/paypal/paypal-checkout-components/compare/chore/demo...chore/demo-no-babel?expand=1. The buttons seemed to load fine for me with it. I made sure to disable cache. We can address later along with the other demos if we end up not needing babel.

**Comment by omaribarra-dev** [comment]
Thank you is there any way to include this in a nuxt component I have already try this but no luck, could you please give me a hint?

**Pull request merged**